obj/login.js

```diff
@@ -0,0 +1,16 @@
+module.exports = {
+    //DOM elements
+    userSelect : ".Select-placeholder",
+    userSelectedValue : ".Select-value-label",
+    userSelectListValue : '[aria-label="?"]',
+    roleSelect : '[id="react-select-3--value"]',
+    roleSelectListValue : '[aria-label="?"]',
+    roleSelectedValue : '[id="react-select-3--value"]',
+    //Functions
+    getUserSelect : function(userName) {
```

Comment:
function name is not descriptive enough and actually misleading as it is getUserSelectValue not whole select